src/validation.h

```diff
@@ -157,12 +157,13 @@ extern uint64_t nPruneTarget;
  * May not be called in a
  * validationinterface callback.
  *
- * @param[in]   pblock  The block we want to process.
- * @param[in]   fForceProcessing Process this block even if unrequested; used for non-network block sources and whitelisted peers.
- * @param[out]  fNewBlock A boolean which is set to indicate if the block was first received via this call
- * @returns     If the block was processed, independently of block validity
+ * @param[in]   pblock            The block we want to process.
+ * @param[out]  state             Currently unused.
```

Comment:
db0eecc

nit: Is there any guidelines on parameters sorting ? You can sort by param[in] then param[out]. It makes easier to reason on function. You can also precise in commit that new `dos_state` isn't used.